### ISSUE TITLE
Client build problem on Windows

### ISSUE THREAD

**Issue by YegorZima** [owncloud/client#7147]
I tried to build ownCloud client on Windows 10.
But it shows this error.
I think there is problem on VFS. Can I disable it or there is another way?
Please help me.

![Untitled](https://user-images.githubusercontent.com/48161507/56067655-4e947680-5dbf-11e9-8a21-6a02fcda26c3.png)

### Client configuration
Client version:
2.6.0
Operating system:
Windows 10
OS language:
MinGW 7.3
Qt version used by client package (Linux only, see also Settings dialog):
5.12.2

Thanks.

**Comment by michaelstingl** [comment]
Did you follow the [Craft instructions](https://github.com/owncloud/client/blob/master/docs/modules/ROOT/pages/building.adoc#windows-development-build-with-kde-craft)?

**Comment by YegorZima** [comment]
Thank you for your reply.
I tried but it has also issue.

![Untitled2](https://user-images.githubusercontent.com/48161507/56079137-994fd600-5e33-11e9-8a99-21a3325df2ac.png)

Can you help me?
At first, can you tell me why vfs issue occured, and if I can disable vfs option in makefile?
Thank you again.

**Comment by michaelstingl** [comment]
@ckamm @dschmidt anything VFS related that needs to be adjusted in the docs or makefile?

**Comment by YegorZima** [comment]
michaelstingl. Thanks for your reply.
I've passed that issue.
There was a issue in cmake. 
I used this one to get client source code.

git clone git://github.com/owncloud/client.git
git checkout master
git submodule update --init --recursive

With this, I can get files in zsync folder and cmake passed successfully without any errors.
And I build it.
And this issue solved.
Thank you.

**Comment by YegorZima** [comment]
But unfortunately I have one more issue.
Anybody can help me?
I want to use client binary files on another computer.
I copied dependency dll files including qt, then I could run.
But when I tried to connect to server, it shows this error.

Timeout while trying to connect to ...

I noticed that it comes from SSL.
On another computer, it doesn't load ssl library, so it can't connect to server, I think.
I set ssl_path before build already.
How can I do next?
Thank you.
